qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -182,6 +182,7 @@
                         fio_options += '--bs=4K --size=1G --iodepth=256 --numjobs=128 --runtime=1800 --thread'
                 - with_pjdfstest:
                     no s390 s390x
+                    no ppc64 ppc64le
```

Comment:
Platforms limitation, suggest merging this into `no s390 s390x`.

style like:
```
no platform_1 platform_2
no os_1 os_2
no machine_type_1 machine_type2
```

Reply:
Updated, could you help to review it ? Thanks a lot